client/my-sites/email/email-providers-comparison/style.scss

```diff
@@ -1,6 +1,44 @@
 @import '@wordpress/base-styles/breakpoints';
 @import '@wordpress/base-styles/mixins';
 
+.action-panel {
+	flex-direction: row;
+
+	&.is-primary {
+		flex-direction: column;
+		@include break-mobile {
+			flex-direction: row;
+		}
+	}
+}
+
+.action-panel__figure {
+	float: right;
+	max-width: 208px;
+
+	&.is-inline-body-text {
+		margin-top: 40px;
+	}
+
+	&.align-left {
+		float: left;
+		margin-left: 0;
+		margin-right: 24px;
+		width: unset;
+	}
+
+	&.align-right {
+		margin: 0 0 24px 24px;
+	}
+
+	&.align-left {
+		float: left;
+		margin-left: 0;
+		margin-right: 24px;
+	}
+}
```

Comment:
@AllTerrainDeveloper, please qualify these styles? Right now, the behaviour will depend on the load order for the relevant CSS files, when we only want these styles to apply to these elements that are within the email comparison component.

Can you make these beneath one of the wrapping CSS classes for the component to make it explicit that these styles should only show up for this page?

Reply:
Good point. I've updated the CSS, and I have not updated screenshots as they are behaving the same as with the previous commit.